apps/tad/overlays/production/kustomization.yaml

```diff
@@ -19,6 +19,15 @@ images:
     newTag: main@sha256:7dbcf8075c4c98e50fcd4ed63d00cec6e228d36678f67ed585cdcda0844cfb93
 
 patches:
+  - target:
```

Comment:
Would it not be easier to just replace the base deployment? The change is valid for production and staging

Reply:
Ok, done.